### PR TITLE
feat: filter trades by ignored

### LINKED ISSUE #5570: Allow to filter trades by ignored/not ignored
## Abstract

Allow users to filter ignored/un ignored trades in the trade page

## Motivation

This is useful when doing changes to the PnL report to check what is ignored and what is not

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #5570

Still need backend fix for some bug. Need to be resolved @prettyirrelevant 
image

### CODE REVIEW

**Comment by kelsos** [comment]
@lukicenturi is the backend PR this is based on merged, maybe we can rebase then

**Comment by lukicenturi** [comment]
> @lukicenturi is the backend PR this is based on merged, maybe we can rebase then

That's backend commit isn't in develop yet.
It is some bugfixes I ask Isaac to add, for the pagination.

File: rotkehlchen/db/filtering.py [hunk 1]
```diff
@@ -646,6 +647,13 @@ def make(
         if location is not None:
             filter_query.location_filter = DBLocationFilter(and_op=True, location=location)
             filters.append(filter_query.location_filter)
+        if trades_idx_to_ignore is not None:
+            filters.append(DBIgnoreValuesFilter(
+                and_op=True,
+                column='id',
+                # only iteration is performed, so converting to a list is not efficient
+                values=trades_idx_to_ignore,  # type: ignore[arg-type]
```
Comment (LefterisJP) [inline rotkehlchen/db/filtering.py:655]: Why not simpy change the type of `DBIgnoreValueFilter(values)` to also allow set?

**Review by LefterisJP — APPROVED** [review]
Approving the backend changes only, after Isaac's changes -- frontend should be checked.

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**